src/main/java/org/mockito/internal/debugging/LocationFactory.java

```diff
@@ -24,26 +24,6 @@ private interface Factory {
     }
 
     private static Factory createLocationFactory() {
```

Comment:
Nit: this whole factory can now be removed and we can use the relevant location directly

Reply:
This is true, thanks @TimvdLippe !